app/stac_api/managers.py

```diff
@@ -153,6 +162,73 @@ def _parse_datetime_query(self, date_time):
             )
         return start, end
 
+    def filter_by_ids(self, ids_array):
+        '''Filter by ids parameter
+
+        Args:
+            ids_array: list[string]
+                An array of ids (string)
+
+        Returns:
+            queryset filtered by a list of ids
+        '''
+        return self.filter(name__in=ids_array)
+
+    def filter_by_collections(self, collections_array):
+        '''Filter by collections parameter
+
+        Args:
+            collections_array: list[string]
+                An array of collections (string)
+
+        Returns:
+            queryset filtered by a list of collections
+        '''
+        return self.filter(collection__name__in=collections_array)
+
+    def filter_by_intersects(self, intersects):
+        '''Filter by intersects parameter
+
+        Args:
+            intersects: string
+                Is a geojson formatted string
+
+        Returns:
+            queryset filtered by intersects
+        '''
+        the_geom = GEOSGeometry(intersects)
+        return self.filter(geometry__intersects=the_geom)
+
+    def filter_by_query(self, query):
+        '''Filter by the query parameter
+
+        Args:
+            query: string
+                The query is a json formatted string.
+
+        Returns:
+            queryset filtered by query
+        '''
+        queriabel_date_fields = ['datetime', 'created', 'updated']
+        json_query = json.loads(query)
```

Comment:
I would do the conversion (and error handling!) of query string to json in the view and provide the `query` as dict parameter here.

Reply:
thanks